firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Protocols/messages.txt

```diff
@@ -0,0 +1,181 @@
+/* Write function
+Scenario: Message: The write method correctly generates an encoded body from the protobuf message descriptor payload
+
+  GIVEN("A Message object constructed with StateSegment Taggedunion, default message types and a max size of 252 bytes")
```

Comment:
Since no explicit constructor is defined for `Message`, I think it's more accurate to say "A Message object default-constructed with ... as its class template parameters". Or you could use this GIVEN to also initialize the values of the `type` and `payload` members.